servers/visual/visual_server_canvas.cpp

```diff
@@ -667,6 +679,8 @@ void VisualServerCanvas::render_canvas(Canvas *p_canvas, const Transform2D &p_tr
 		memset(z_list, 0, z_range * sizeof(RasterizerCanvas::Item *));
 		memset(z_last_list, 0, z_range * sizeof(RasterizerCanvas::Item *));
 
+		_current_camera_transform = p_transform;
```

Comment:
### Note
This is to avoid passing the camera transform down the rendering function calls, as it won't be used in 99% of cases, because the camera transform is normally concatenated with the item global transform. It should be invariant throughout the call (I hope :wink: ). If this assumption proves incorrect we can pass it down or take different approach.